kiwi_lint/generic_foreign_key.py

```diff
@@ -0,0 +1,34 @@
+import astroid
+from pylint.checkers import utils, BaseChecker
+from pylint.interfaces import IAstroidChecker
+from pylint_django.utils import node_is_subclass
+
+
+class GenericForeignKeyChecker(BaseChecker):
+    __implements__ = (IAstroidChecker,)
+
+    name = 'generic-foreign-key-checker'
+
+    msgs = {
+        'R4525': (
```

Comment:
ID number needs to change but that depends on which one of the other open PRs gets merged first.

Reply:
Updated